apis/infix/atrium-api-infix/src/commonTest/kotlin/ch/tutteli/atrium/api/infix/en_GB/IterableToContainInAnyOrderOnlyValuesExpectationsTest.kt

```diff
@@ -43,21 +44,21 @@ class IterableToContainInAnyOrderOnlyValuesExpectationsSpec :
 
 
     @Suppress("unused", "UNUSED_VALUE")
-    private fun ambiguityTest() {
-        var list: Expect<List<Number>> = notImplemented()
-        var nList: Expect<Set<Number?>> = notImplemented()
-        var subList: Expect<ArrayList<Number>> = notImplemented()
-        var star: Expect<Collection<*>> = notImplemented()
+    @Test
+    fun ambiguityTest() {
+        var list: Expect<List<Number>> = expect(listOf(1))
+        var nList: Expect<Set<Number?>> = expect(setOf(1))
+        var subList: Expect<ArrayList<Number>> = expect(listOf(1))
+        var star: Expect<Collection<*>> = expect(listOf(1))
 
         list = list toContain o inAny order but only value(1)
         nList = nList toContain o inAny order but only value(1)
         subList = subList toContain o inAny order but only value(1)
         star = star toContain o inAny order but only value(1)
 
-        //TODO type parameter should not be necessary, check with Kotlin 1.4
-        list = list toContain o inAny order but only the values<Number>(1, 1.2)
-        nList = nList toContain o inAny order but only the values<Number>(1, 1.2)
-        subList = subList toContain o inAny order but only the values<Number>(1, 2.2)
+        list = list toContain o inAny order but only the values(1, 1.2)
```

Comment:
e.g. this one will fail since 1.2 is not in there. You need to add a `var listValues = ... expect(listOf(1, 1.2)` nList=... etc. so that you can cover both `value` and `values`